image/image.go

```diff
@@ -249,44 +249,21 @@ func NewChildImage(img *Image, child ChildConfig, os string) *Image {
 }
 
 // History stores build commands that were used to create an image
-type History struct {
-	// Created is the timestamp at which the image was created
-	Created time.Time `json:"created"`
-	// Author is the name of the author that was specified when committing the
-	// image, or as specified through MAINTAINER (deprecated) in the Dockerfile.
-	Author string `json:"author,omitempty"`
-	// CreatedBy keeps the Dockerfile command used while building the image
-	CreatedBy string `json:"created_by,omitempty"`
-	// Comment is the commit message that was set when committing the image
-	Comment string `json:"comment,omitempty"`
-	// EmptyLayer is set to true if this history item did not generate a
-	// layer. Otherwise, the history item is associated with the next
-	// layer in the RootFS section.
-	EmptyLayer bool `json:"empty_layer,omitempty"`
-}
+type History = ocispec.History
```

Comment:
Do we want to keep the alias? If we want it to go away eventually, we could;

```suggestion
//
// Deprecated: use [ocispec.History] instead.
type History = ocispec.History
```

Reply:
Up for deleting it completely! Even considering consumers, I think it's a pretty straightforward change.

Reply:
We should replace uses in our code, but I'm for "keeping it (as deprecated)" for 1 release; that way consumers will (hopefully) get a deprecation warning from their linters, and the link gives a clue how to resolve it. See https://github.com/moby/moby/issues/45707 as an example (unfortunately there no linter warned about it though, but usually it would).